src/game/objects/traps/movable_block.c

```diff
@@ -22,28 +22,30 @@ typedef enum {
     MBS_PULL = 3,
 } MOVABLE_BLOCK_STATE;
 
-static int16_t m_MovingBlockBounds[12] = {
-    -300,
-    +300,
-    0,
-    0,
-    -WALL_L / 2 - (LARA_RAD + 80),
-    -WALL_L / 2,
-    -10 * PHD_DEGREE,
-    +10 * PHD_DEGREE,
-    -30 * PHD_DEGREE,
-    +30 * PHD_DEGREE,
-    -10 * PHD_DEGREE,
-    +10 * PHD_DEGREE,
+static const OBJECT_BOUNDS m_MovableBlock_Bounds = {
+    .shift = {
+        .min = { .x = -300, .y = 0, .z = -WALL_L / 2 - (LARA_RAD + 80), },
+        .max = { .x = +300, .y = 0, .z = -WALL_L / 2, },
+    },
+    .rot = {
+        .min = { .x = +10 * PHD_DEGREE, .y = +30 * PHD_DEGREE, .z = +10 * PHD_DEGREE, },
+        .max = { .x = -10 * PHD_DEGREE, .y = -30 * PHD_DEGREE, .z = -10 * PHD_DEGREE, },
```

Comment:
I can't move movable blocks. I think these min and max are reversed?

Reply:
Good catch, I've fixed this.